.gitignore

```diff
@@ -0,0 +1,2 @@
+/package-lock.json
```

Comment:
Instead of ignoring `package-lock.json` I personally prefer to create a `.npmrc` file with `package-lock=false`. That way a stale package-lock.json file on your own computer (which will then be ignored) can't prevent you from using the latest versions of your dependences...

Reply:
done